README-KO.md

```diff
@@ -148,25 +148,25 @@ Scala는 매우 강력하며 여러가지 페러다임에 적용 가능한 언
 
 일반적으로:
 
-- 함수는 30줄 이상의 라인을 초과하지 않아야 합니다.
-- 하나의 클래스당 30개 이상의 함수를 갖지 않도록 합니다.
+- 메소드는 30줄 이상의 라인을 초과하지 않아야 합니다.
+- 하나의 클래스당 30개 이상의 메소드를 갖지 않도록 합니다.
 
 
 ### <a name='indent'>공백 및 들여쓰기</a>
 
-- 연산자 및 할당 연산자 앞 뒤에는 1칸 공백을 두도록 합니다.
+- 연산자 및 할당 연산자 앞 뒤에는 1칸 스페이스를 두도록 합니다.
   ```scala
   def add(int1: Int, int2: Int): Int = int1 + int2
   ```
 
-- 콤마 뒤에는 1칸 공백을 두도록 합니다.
+- 콤마 뒤에는 1칸 스페이스를 두도록 합니다.
   ```scala
   Seq("a", "b", "c") // 이와 같이 하도록 합니다.
 
-  Seq("a","b","c") // 콤마 뒤에는 공백을 생략히지 않습니다.
+  Seq("a","b","c") // 콤마 뒤에는 공백을 생략하지 않습니다.
```

Comment:
This non-breaking space actually breaks md rendering:

<img width="430" alt="2017-10-28 5 18 43" src="https://user-images.githubusercontent.com/6477701/32132728-132ec9aa-bc04-11e7-9c2d-8321be98bdb7.png">

```
>>> """  Seq"""
' \xc2\xa0Seq'
```

I submitted a PR to Spark before. Please refer `https://github.com/apache/spark/pull/17517` for more info.